build-system/eslint-rules/no-forbidden-terms.js

```diff
@@ -33,7 +35,9 @@ module.exports = function (context) {
         const {match, message, loc} = report;
         context.report({
           loc,
-          message: `Forbidden: "${match}".${message ? `\n${message}.` : ''}`,
+          message:
+            `Forbidden: "${match}".` +
```

Comment:
It might make the message easier to read if you color-format the match.
```suggestion
            `Forbidden: ${formattedMatch}.` +
```

Reply:
Nice, VSCode appears to be reasonable and strips out the color codes.